### PR TITLE
CSS Modifications

### PR DESCRIPTION
**Pull request by DiceT**
Some changes to the Starforged theme to flesh out the sci-fi theme and add some contrast to various items.

- [ ]
- [ ] Update CHANGELOG.md

### CODE REVIEW

File: src/module/vue/sf-locationsheet.vue [hunk 1]
```diff
@@ -112,7 +112,7 @@
   padding: 7px;
 }
 .highlighted {
-  background: lightyellow;
+  background: #33999933;
```
Comment (ben) [inline src/module/vue/sf-locationsheet.vue:115]: When I first saw this, I thought "that can't possibly work in the standard black-and-white theme" but actually it kind of does.
CleanShot 2022-04-24 at 19 05 23
CleanShot 2022-04-24 at 19 05 37

**Comment by ben** [comment]
This is definitely an improvement, and I like it a lot. I'm going to go ahead and merge it, and I'll pop something into the changelog and ship a release.

CleanShot 2022-04-24 at 19 07 52

File: src/module/vue/sf-locationsheet.vue (hunk 1 above)
Comment (DiceT) [reply src/module/vue/sf-locationsheet.vue:115]: It actually does look good. Did you add it to the Ironsworn style? Or is vue file used in both?

**Comment by DiceT** [comment]
> This is definitely an improvement, and I like it a lot. I'm going to go ahead and merge it, and I'll pop something into the changelog and ship a release.

I will have to really dig in there eventually. Would love to get all the results and links themed out consistently.
 
image

**Pull request merged**